ContactList/ContactList/ViewControllerForShow.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  ViewController.swift
+//  ContactList
+//
+//  Created by Maksym Humeniuk on 5/6/19.
+//  Copyright © 2019 Maksym Humeniuk. All rights reserved.
+//
+
+import UIKit
+
+class ViewControllerForShow: UIViewController {
+    
+    
+    @IBOutlet weak var firstName: UILabel!
+    
+    @IBOutlet weak var lastName: UILabel!
+    
+    @IBOutlet weak var phoneNumber: UILabel!
+    
+    @IBOutlet weak var email: UILabel!
+    
+    @IBOutlet weak var imageArea: UIImageView!
+    
+    var person: Person!
+    
+    var currentAttayIndex: Int!
```

Comment:
You should not operate indexPath/index.
Add `contactId: String` into `Contact` and operate it.
Use `UUID().uuidString`